docs/UserGuide.adoc

```diff
@@ -385,7 +385,7 @@ This updates the restaurants to cater for a capacity for 50 customers
 
 Creates a new booking.
 
-Format: `addBooking [c/CUSTOMER_ID] n/NUMBER OF PERSONS t/TIME [d/DATE]`
+Format: `addBooking [c/CUSTOMER_ID] n/NUMBER OF MEMBERS t/TIME [d/DATE]`
```

Comment:
lol i think i want to leave this as persons? i mean when people make a restaurant booking you ask how many people right